src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/NotInstruction.cs

```diff
@@ -2,187 +2,191 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Do not edit this file manually. This file is auto-generated by NotInstruction.tt.
+
+using System.Diagnostics;
 using System.Dynamic.Utils;
+using System.Reflection;
 
 namespace System.Linq.Expressions.Interpreter
 {
     internal abstract class NotInstruction : Instruction
     {
-        public static Instruction _Bool, _Int64, _Int32, _Int16, _UInt64, _UInt32, _UInt16, _Byte, _SByte;
-
-        private NotInstruction() { }
+        private static Instruction s_Boolean, s_SByte, s_Int16, s_Int32, s_Int64, s_Byte, s_UInt16, s_UInt32, s_UInt64;
 
         public override int ConsumedStack => 1;
         public override int ProducedStack => 1;
         public override string InstructionName => "Not";
 
-        private sealed class BoolNot : NotInstruction
+        private NotInstruction() { }
```

Comment:
This file was very inconsistent compared to other unary operators, hence the larger diff.